src/coverlet.msbuild.tasks/coverlet.msbuild.tasks.csproj

```diff
@@ -2,20 +2,62 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net472</TargetFrameworks>
     <AssemblyVersion>2.4.1</AssemblyVersion>
+
+    <PackageId>coverlet.msbuild</PackageId>
+    <PackageVersion>2.5.1</PackageVersion>
+    <Title>coverlet.msbuild</Title>
+    <Authors>tonerdo</Authors>
+    <PackageLicenseUrl>https://github.com/tonerdo/coverlet/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>http://github.com/tonerdo/coverlet</PackageProjectUrl>
+    <PackageIconUrl>https://raw.githubusercontent.com/tonerdo/coverlet/master/_assets/coverlet-icon.svg?sanitize=true</PackageIconUrl>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <DevelopmentDependency>true</DevelopmentDependency>
+    <Description>Coverlet is a cross platform code coverage library for .NET Core, with support for line, branch and method coverage.</Description>
+    <PackageTags>coverage testing unit-test lcov opencover quality</PackageTags>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+
+    <!-- Items in the Content group are placed in this folder in the NuGet package. -->
+    <ContentTargetFolders>build</ContentTargetFolders>
+
+    <!-- Build tasks should not be added to the lib folder. -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSources>true</IncludeSources>
```

Comment:
Any reason we're including symbols and sources?

Reply:
It can improve the debugging experience if needed. We can alter the way this is included in a future PR if desired.